consumer.go

```diff
@@ -415,36 +413,59 @@ func (c *Consumer) fetchSubscriprionWithTimeout() ([]*Msg, error) {
 	}()
 	select {
 	case <-time.After(timeoutDuration):
-		return nil, memphisError(errors.New("fetch timed out"))
+		return nil, nil
```

Comment:
why nil in here? it seems like a timeout

Reply:
you are right, forgot it